### DISCUSSION TITLE
[Desktop] How to draw Control using Skia?

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno#16908] [Q&A] [answered]
I am reconsidering to create controls somehow similar to [livecharts](https://livecharts.dev/)
Could you please provide simple code snippet that creates a (user?) control that performs direct drawing using Skia?
Shall work on Uno Desktop apps (win/linux/macos).

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. You can check this sample on how to use SkiaSharp: https://github.com/unoplatform/Uno.Samples/tree/master/UI/SkiaSharpTest.
